### PR TITLE
fixed nested issue

### PR DESCRIPTION
**Pull request by beahoop**
## Types of changes

- [ x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/components/FormField/FormField.tsx [hunk 1]
```diff
@@ -72,9 +73,11 @@ export const FormField = React.forwardRef(({
             key = 3;
         }
 
+        const isError = _.get(errors, `${name}[message]`)
         /** if there is an error, return that */
-        if (errors[name]?.message) {
-            content = errors[name]?.message;
+        if (isError) {
+            console.log(isError)
```
Comment (ItsEricWoodward) [inline src/components/FormField/FormField.tsx:79]: I think you forgot to remove this.

**Review by ItsEricWoodward — CHANGES_REQUESTED** [review]
👍 apart from the `console.log`. Excise that and you'll be good to go.

File: src/components/FormField/FormField.tsx [hunk 2]
```diff
@@ -104,7 +107,7 @@ export const FormField = React.forwardRef(({
      */
     if (Component === TextField) return (
         <Component
-            error={Boolean(errors[name])}
+            error={_.get(errors, `${name}[message]`)}
```
Comment (emdrayallday) [inline src/components/FormField/FormField.tsx:110]: This isn't right, what happens when there isn't a message but still an error?

File: src/components/FormField/FormField.tsx [hunk 3]
```diff
@@ -3,6 +3,7 @@ import { useFormContext } from 'react-hook-form';
 import FormControlLabel from '@material-ui/core/FormControlLabel';
 import { AnimatePresence, motion } from 'framer-motion';
 import Checkbox from '../Checkbox';
+import _ from 'lodash';
```
Comment (emdrayallday) [inline src/components/FormField/FormField.tsx:6]: destructure

**Review by emdrayallday — CHANGES_REQUESTED** [review]
Close but not quite.

you should probably name the error message better if it has one. isError doesn't really make sense to me. I would just name it errorMessage.

then for the error attribute on the <Component you need to check if there is an error, not an error message

File: src/components/FormField/FormField.tsx [hunk 4]
```diff
@@ -72,9 +73,11 @@ export const FormField = React.forwardRef(({
             key = 3;
         }
 
+        const isError = _.get(errors, `${name}[message]`);
```
Comment (emdrayallday) [inline src/components/FormField/FormField.tsx:76]: also instead of [message] do `${name}.message`

**Review by emdrayallday — APPROVED** [review]

**Pull request merged**